File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{

float change = -1;
int coins = 0;

while(change < 0)

    {
    change = get_float("How much change is needed?");
    }

    int cash = round(change * 100);

    if(cash >= 25)
    {
        cash -= 25;
        coins += 1;
    }
    if(cash >= 10)
    {
        cash -= 10;
        coins += 1;
    }
    if(cash >= 5)
    {
        cash -= 5;
        coins += 1;
    }
    else
    {
        cash -= 1;
        coins += 1;
    }
    printf("%i", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

money = int(round(change * 100))
coins = 0

for i in [25, 10, 5, 1]:
    coins += money // i
    money = money % i

print(coins)